content/docs/04-clusters/03-edge/02-edge-configuration/35-installer-reference.md

```diff
@@ -155,6 +155,52 @@ Network settings specific to the network interface of the edge host. You can con
 | `networkInterface.gateway` | The network gatway IP address. |
 | `networkInterface.nameserver` | The IP address of the DNS nameserver this interface should route requests to.| 
 
+## User ID (UID) Parameters
+
+The User ID(UID) is generated by a specific priority sequence. The below table outlines the priority order from top to bottom, to generate a UID. The UID generation starts with priority 1, the device `name`, followed by attributes within the `deviceUID`, then obtaining the UID from the `/sys/class/dmi/id/product_uuid` file, and finally generating a random UUID if all other methods are unsuccessful.
```

Comment:
```suggestion
The device ID is generated by a specific priority sequence. The below table outlines the priority order from top to bottom when generating a UID for the Edge host. The UID generation starts with priority one, the device `name`, followed by attributes within the `deviceUID`, then obtaining the UID from the **/sys/class/dmi/id/product_uuid** file, and finally generating a random UUID if all other methods are unsuccessful.
```
Let's simplify the language in this paragraph, it's a bit tough to read. File paths are also in bold per our style guide.